lib/lighthouse/benefits_claims/service.rb

```diff
@@ -35,6 +37,29 @@ def get_intent_to_file(type, lighthouse_client_id = nil, lighthouse_rsa_key_path
       handle_error(e, lighthouse_client_id, endpoint)
     end
 
+    # For type "survivor", the request must include claimantSsn and be made by a valid Veteran Representative.
+    # If the Representative is not a Veteran or a VA employee, this method is currently not available to them,
+    # and they should use the Benefits Intake API as an alternative.
+    def create_intent_to_file(type, lighthouse_client_id, lighthouse_rsa_key_path, options = {})
+      endpoint = 'benefits_claims/intent_to_file'
+      path = "#{@icn}/intent-to-file"
+      response = config.post(
```

Comment:
yeah, `response` variable isn't used anywhere. can probably remove the variable declaration :)